admin_manual/release_notes.rst

```diff
@@ -13,6 +14,44 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.5_release_notes_label:
+
+Changes in 10.0.5
+-----------------
+
+Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.0.5 that need your attention. You can also read `the full ownCloud Server changelog`_ for further details on what has changed.
+
+Technology preview for PHP 7.2 support
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+ownCloud catches up with new web technologies. This has mainly been introduced for the open-source community to test and give feeback. PHP 7.2 is not yet supported nor recommended for productive scenarios. ownCloud is going to fully support PHP 7.2 with the next major release.
+
+php-intl now is a hard requirement
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Please make sure to have the PHP extension installed before upgrading.
+
+Changed: Only allow a single active theme app
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The theming behavior has been changed for that only a single theme can be active concurrently. This change ensures that themes can not interfere in any way. Please make sure to have the desired theme enabled after upgrading.
```

Comment:
s/for that/so that/

Reply:
> can not interfere in any way

Can you expand on this?

Reply:
themes app usually override core stuff. if you had two enabled then two apps would be overriding stuff in an arbitrary order